subsys/bluetooth/controller/ll_sw/ull_sync_iso.c

```diff
@@ -521,6 +524,22 @@ void ull_sync_iso_setup(struct ll_sync_iso_set *sync_iso,
 	}
 	ticks_slot_offset += HAL_TICKER_US_TO_TICKS(EVENT_OVERHEAD_START_US);
 
+	/* Check and skip to next interval if CPU processing introduces latency
+	 * that can delay scheduling the first ISO event.
+	 */
+	ticks_expire = ftr->ticks_anchor - ticks_slot_offset +
+		       HAL_TICKER_US_TO_TICKS(sync_iso_offset_us);
+	ticks_threshold = ticker_ticks_now_get() +
+			  HAL_TICKER_US_TO_TICKS(EVENT_OVERHEAD_START_US);
+	ticks_diff = ticker_ticks_diff_get(ticks_expire, ticks_threshold);
+	if (ticks_diff & BIT(HAL_TICKER_CNTR_MSBIT)) {
+		sync_iso_offset_us += interval_us -
+			lll->window_widening_periodic_us;
+		lll->window_widening_event_us +=
+			lll->window_widening_periodic_us;
+		lll->payload_count += lll->bn;
+	}
```

Comment:
Any risks of underflow or overflow on any of these?

Reply:
`ticks_*` are permited to underflow and overflow as it is free-running. `window_widening_periodic_us` is a fraction of `interval_us` hence would not lead to negative offset.